src/sentry/testutils/pytest/relay.py

```diff
@@ -81,8 +81,8 @@ def relay_server_setup(live_server, tmpdir_factory):
     template_vars = {
         "SENTRY_HOST": f"http://host.docker.internal:{port}/",
         "RELAY_PORT": relay_port,
-        "KAFKA_HOST": "kafka-kafka-1" if use_new_dev_services else "sentry_kafka",
-        "REDIS_HOST": "redis-redis-1" if use_new_dev_services else "sentry_redis",
+        "KAFKA_HOST": "sentry_kafka" if use_old_devservices else "kafka-kafka-1",
```

Comment:
We should actually be able to use just `kafka` and `redis` instead of `kafka-kafka-1` and `redis-redis-1` here, since the containers have two aliases based on container name and service name used in docker compose

Reply:
I feel like it may be better to keep it consistent with how everything else looks, but we can change it if you think it is worthwhile

Reply:
don't worry about it, due to order of merges this should be picked up